src/plugins/workspace/public/components/workspace_form/workspace_permission_setting_input.tsx

```diff
@@ -175,7 +174,7 @@ export const WorkspacePermissionSettingInput = ({
           placeholder={
             type === WorkspacePermissionItemType.User
               ? i18n.translate('workspaceForm.permissionSetting.selectUser', {
-                  defaultMessage: 'Enter user name or uer ID',
+                  defaultMessage: 'Enter user name or email address',
```

Comment:
Currently we do not support email address, I think we should still use `user ID`.

Reply:
ok

Reply:
ok I‘ll fix it in this PR https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8383/